app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,887 @@
+
+    '''
+
+    with open(cfg_in.run_config_path, 'r+', encoding='utf8') as fin:
+        runconfig_dict_in = yaml.safe_load(fin.read())
+
+    list_runconfig_burst = []
+
+    # determine the bursts to process
+    list_burst_id = cfg_in.bursts.keys()
+
+    # determine the output directory for child process
+    for burst_id in list_burst_id:
+        path_temp_runconfig = os.path.join(cfg_in.scratch_path,
+                                           f'burst_runconfig_{burst_id}.yaml')
+
+        runconfig_dict_out = runconfig_dict_in.copy()
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'input_file_group',
+                                 'burst_id'],
+                                [burst_id])
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'processing',
+                                 'geocoding',
+                                 'memory_mode'],
+                                'single_block')
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'output_dir'],
+                                 output_dir_child)
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                    ['runconfig',
+                                    'groups',
+                                    'product_group',
+                                    'scratch_path'],
+                                    os.path.join(cfg_in.groups.product_group.scratch_path,
+                                                 f'{os.path.basename(output_dir_child)}_child_scratch')
+                                    )
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'save_secondary_layers_as_hdf5'],
+                                 False)
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'output_imagery_format'],
+                                 'GTiff')
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'save_mosaics'],
+                                False)
+
+        # TODO Think if it would be necessary to always turn on `save_bursts` for the child runconfigs?
```

Comment:
Yes, you need to set `save_bursts` to `True` for the child ruconfigs. Please, update the code and remove this TODO.